xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/MMALRenderer.cpp

```diff
@@ -821,6 +823,9 @@ void CMMALRenderer::RenderUpdate(bool clear, DWORD flags, DWORD alpha)
   int source = m_iYV12RenderBuffer;
   CMMALBuffer *omvb = nullptr;
 
+  CRenderingStats stats = g_windowManager.GetRenderingStats();
```

Comment:
this thread will finally call Windowing::PresentRender which can be overwritten by the Pi port. then there is no need to have g_Windowing.SetVisible